packages/wallet/ui/src/components/BrandIcon.jsx

```diff
@@ -0,0 +1,16 @@
+import Box from '@mui/material/Box';
+import { icons, defaultIcon } from '../util/Icons.js';
+
+const BrandIcon = ({ brandPetname }) => {
```

Comment:
it's just a `name`, right? petnames are arbitrary per user and this is keyed from source control